pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -280,7 +280,7 @@ func (ed *eventDispatcher) Unregister(ctx context.Context, request *lifecycle.Un
 	return &lifecycle.UnregisterReply{}, nil
 }
 
-func eventReplyValidator(reply *lifecycle.EventReply) (bool, error) {
+func isEventReplyValid(reply *lifecycle.EventReply) (bool, error) {
 	if reply == nil {
 		glog.Info("eventDispatcherNoop has been detected - skipping container configuration update")
```

Comment:
This seems a little fishy, no? Ensuring that reply == nil is only ever caused by noop dispatcher might be difficult to guarantee as the code changes. Perhaps we could move this log to the noop event dispatcher callbacks and then return an empty EventReply from there? That way reply == nil is always simply an error we don't expect to occur.